ts/src/renderer/phaser/classes/PhaserChatBubble.ts

```diff
@@ -1,55 +1,40 @@
 class PhaserChatBubble extends Phaser.GameObjects.Container {
 
 	private readonly triangle: Phaser.GameObjects.Graphics;
-	private readonly bubble: Phaser.GameObjects.Graphics;
-	private readonly text: Phaser.GameObjects.Text;
+	protected bubble: Phaser.GameObjects.Graphics;
+	protected textObject: Phaser.GameObjects.Text;
```

Comment:
test whether this change has an effect. I don't think it did as we no longer adjust textObject `alpha`